### ISSUE TITLE
Highlight all buildings that are not building=yes

### ISSUE THREAD

**Issue by RAytoun** [osmlab/applepaintstyles#14]
We often get some individual that feels that building=yes is not enough and embellishes buildings with other 
attributes such as house or hut or industrial. Is it possible to highlight any buildings that are not building=yes?

![Not building=yes](https://user-images.githubusercontent.com/10500102/96311047-8279d900-1000-11eb-85a7-66978853d66d.jpg)

**Comment by wireguy** [comment]
You could also use josm's find with the string:
-building=yes building=*
(no building=yes but all building tags)

and then create a toolbar button

**Comment by westlakej** [comment]
Hey @RAytoun - We added this new setting in the [v1.3 release](https://github.com/osmlab/applepaintstyles/releases/tag/v1.3) which is now live.

As mentioned in a different issue, we broke out the building validation settings into its own group for easier access. To enable this setting right-click on the paint style name > "Style settings" > "Building Validation Settings" > "Building!=yes" 

<img width="582" alt="Screen Shot 2021-01-14 at 9 52 53 AM" src="https://user-images.githubusercontent.com/20782296/104633103-834bf680-5653-11eb-871b-498dcddfc097.png">

When this setting is enabled, all buildings with values other than "yes" will highlight: 

<img width="325" alt="Screen Shot 2021-01-14 at 10 31 44 AM" src="https://user-images.githubusercontent.com/20782296/104633239-b8584900-5653-11eb-903d-3dd9d8d511b9.png">

Thanks for the idea and happy mapping!
